atomsci/ddm/pipeline/model_pipeline.py

```diff
@@ -20,10 +20,18 @@
 import pandas as pd
 import pdb
 
-# import this to get atomsci.ddm.__version__
-import atomsci.ddm.pipeline.model_datasets
-
 from atomsci.ddm.utils import datastore_functions as dsf
+if 'AMPL' in dsf.__file__: # install_dev.sh points to github directory
```

Comment:
This should work as long as the user doesn't rename the directory when he/she git clone the repo. I renamed the directory and my mp.__file__ looks like `/usr/WS2/shi12/ampl/master/atomsci/ddm/pipeline/model_pipeline.py`. Maybe we can try `if 'atomsci/ddm/' in dsf.__file__`?

Reply:
install.sh installation:
`/usr/local/lib/python3.7/site-packages/atomsci/ddm/utils/datastore_functions.py`

install_dev.sh installation:
`/g/g16/apaulson/github_repos/AMPL/atomsci/ddm/utils/datastore_functions.py`

Can you think of a time when ampl would not be in site-packages? we could switch the if/then and search for `site-packages` if that would work?